dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -34,26 +35,64 @@ const spaces = / /g;
 const ophanComponentId = (name: string) =>
 	name.toLowerCase().replace(spaces, '-');
 
+const getMerchHighPosition = (
+	collectionCount: number,
+	isNetworkFront: boolean | undefined,
+) => {
+	if (collectionCount < 4) {
+		return 2;
+	} else if (isNetworkFront) {
+		return 5;
+	} else {
+		return 4;
+	}
+};
+
 const shouldInsertMerchHigh = (
 	index: number,
-	isNetworkFront: boolean | undefined,
-	collectionCount: number,
 	isPaidContent: boolean | undefined,
+	collectionCount: number,
+	isNetworkFront: boolean | undefined,
 ) => {
 	const minContainers = isPaidContent ? 1 : 2;
 	if (collectionCount < minContainers) {
 		return false;
 	}
-	let desiredPosition;
-	if (collectionCount < 4) {
-		desiredPosition = 2;
-	} else if (isNetworkFront) {
-		desiredPosition = 5;
-	} else {
-		desiredPosition = 4;
-	}
 
-	return index === desiredPosition;
+	return index === getMerchHighPosition(collectionCount, isNetworkFront);
+};
+// On mobile, we remove the first container if it is a thrasher
+// and remove a container if it, or the next sibling, is a commercial container
+// we also exclude any containers that are directly before a thrasher
+// then we take every other container, up to a maximum of 10, for targeting MPU insertion
+
+const mobileAddPosition: number[] = [];
```

Comment:
We probably don't want to be defining this array at the root level of the document, see my other comments